src/Components/skills.js

```diff
@@ -0,0 +1,179 @@
+import React, { useEffect } from "react";
+import { Col, Container } from "react-bootstrap";
+import ProgressBar from "react-bootstrap/ProgressBar";
+import aos from "aos";
+import "aos/dist/aos.css";
+
+function Skills() {
+  useEffect(() => {
+    aos.init({ duration: 1500 });
+  }, []);
+  const 
+    expert = 85,
+    basics = 33,
+    intermediate = 50,
+    advanced = 70;
+  return (
+    <Container
+      className="centerDiv"
+      id="skills"
+      style={{
+        color: "#20C20E",
+        fontFamily: "Share Tech Mono",
+      }}
+    >
+      <h1
+        style={{
+          fontFamily: "Rubik Mono One",
+          fontSize: "max(3.0vw, 22pt)",
+        }}
+      >
+        3.Professional Skills
+      </h1>
+      <Container className="centerDiv" id="skillswrap">
+        <Col style={{ width: "100%" }}>
+          <label
+            style={{
+              fontSize: "max(1.5vw, 27px)",
+              fontWeight: "bold",
+            }}
+          >
+            Javascript
+          </label>
+          <ProgressBar style={{ height: "2.3vh" }}>
+            <ProgressBar
+              data-aos="zoom-in-right"
+              now={advanced}
+              label={"ADVANCED"}
+              className="advanced"
+              style={{ fontSize: "1.4vh", color: "black", fontWeight: "bold" }}
+            />
+          </ProgressBar>
+          <label
+            style={{
+              fontSize: "max(1.5vw, 27px)",
+              fontWeight: "bold",
+            }}
+          >
+            HTML && CSS
+          </label>
+          <ProgressBar style={{ height: "2.3vh" }}>
+            <ProgressBar
```

Comment:
Also very repetitive here, I think would be better to create some objects in an array detailing the dynamic datapoints and then using a map to generate the HTML.